qdrant_client/http/models/models.py

```diff
@@ -1031,7 +1079,10 @@ class OptimizersConfig(BaseModel, extra="forbid"):
         description="Maximum size (in kilobytes) of vectors allowed for plain index, exceeding this threshold will enable vector indexing  Default value is 20,000, based on &lt;https://github.com/google-research/google-research/blob/master/scann/docs/algorithms.md&gt;.  To disable vector indexing, set to `0`.  Note: 1kB = 1 vector of size 256.",
     )
     flush_interval_sec: int = Field(..., description="Minimum interval between forced flushes.")
-    max_optimization_threads: int = Field(..., description="Maximum available threads for optimization workers")
+    max_optimization_threads: Optional[int] = Field(
```

Comment:
```
    __pydantic_self__.__pydantic_validator__.validate_python(data, self_instance=__pydantic_self__)
pydantic_core._pydantic_core.ValidationError: 1 validation error for OptimizersConfig
max_optimization_threads
  Input should be a valid integer [type=int_type, input_value=None, input_type=NoneType]
  ```

Reply:
We're getting this error on `qdrant-client = "^1.7.3"`;

Is it something expected?

Reply:
do you use dev build?

Reply:
Ah yess, that's the issue; 

ARG QDRANT_VERSION=dev

FROM qdrant/qdrant:${QDRANT_VERSION}

Which one should I use with 1.7.3 client?

Reply:
latest (1.7.4)

Reply:
we have some backward compatibility issue (dev is not backward compatible atm), but we have already made required fixes (but haven't merged them yet)